login.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
```

Comment:
Why is this in login.html instead of index.html?

Reply:
great question

Reply:
didnt want to create merge conflicts lol

Reply:
we can integrate once youre done

Reply:
fair we can merge them after